requirements/requirements.txt

```
@@ -6,13 +6,13 @@ flax>=0.8.1
 gigastep @ git+https://github.com/mlech26l/gigastep
 gymnasium
 hydra-core==1.3.2
-id-marl-eval @ git+https://github.com/instadeepai/marl-eval
+id-marl-eval
```

Since marl-eval hasn't had a release since recent changes can we please keep it as is. 
```suggestion
id-marl-eval @ git+https://github.com/instadeepai/marl-eval
```

---

It has, I did a release about a week ago

---

Is it about the `JsonLogger`? It's working in mava, is there another?

---

ah sorry I do mean matrax :facepalm: